packages/functional-tests/pages/login.ts

```diff
@@ -228,24 +228,28 @@ export class LoginPage extends BaseLayout {
     await this.submit();
   }
 
-  async isEmailHeader() {
-    return this.page.locator(selectors.EMAIL_HEADER).isVisible();
+  async waitForEmailHeader() {
+    await this.page.waitForSelector(selectors.EMAIL_HEADER, {
+      timeout: 2000,
```

Comment:
It might be nice to hold this timeout as constant, since it's repeated a few times. This would also let us easily increase / decrease the value as needed.

Reply:
That's right but there were a lot of places where we had all sort of different values for timeouts(2000ms, 100ms, 1000ms) in this file because of which I didn't use a const. Do you think using const for all these different values would be much easier?